clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/ChainResolver.java

```diff
@@ -128,8 +131,9 @@ public Chain compactChain(Chain chain, long now) {
    * @param key cache key
    * @param existing current state
    * @param operation operation to apply
-   * @param now current time
    * @return an equivalent put operation
    */
-  public abstract PutOperation<K, V> applyOperation(K key, PutOperation<K, V> existing, Operation<K, V> operation, long now);
+  public abstract PutOperation<K, V> applyOperation(K key, PutOperation<K, V> existing, Operation<K, V> operation);
+
+  protected abstract ValueHolder<V> createValueHolder(PutOperation<K, V> operation);
```

Comment:
Missing javadoc

Reply:
Fixed... by refactoring and eliminating the method.